pkg/compliance/clustercompliancereport_test.go

```diff
@@ -216,3 +216,28 @@ func getDetailReport(ctx context.Context, namespaceName types.NamespacedName, cl
 	}
 	return &report, nil
 }
+
+func getStarboardConfig() (starboard.ConfigData, error) {
+	clientset := fakesimple.NewSimpleClientset(
```

Comment:
Why do you use fake here?? ConfigData is a Go struct that you can just instantiate easily.

```go
func getStarboardConfig() (starboard.ConfigData, error) {
	return starboard.ConfigData{
		"compliance.failEntriesLimit": "1",
	}, nil
}
```

You probably don't even need this private method and error in return param.